.github/workflows/typos.yml

```
@@ -0,0 +1,18 @@
+name: Check typos
+
+on:
+  push:
+  pull_request:
+    branches:
+      - '**'
+  workflow_dispatch:
```

```suggestion
  workflow_dispatch:
```
Let's just run it manually for a week to test :)